Upstream: No.
Source: RedHat (https://bugzilla.redhat.com/show_bug.cgi?id=629740)
Reason: systemd needs acpid to fork, but it doesn't when invoked by the
init process

This is a systemd dependend hack so please check upon a version bump
if there has been a proper patch to circumvent this issue.

diff -Naur acpid-2.0.6.orig//acpid.c acpid-2.0.6/acpid.c
--- acpid-2.0.6.orig//acpid.c	2010-03-28 15:54:23.000000000 +0200
+++ acpid-2.0.6/acpid.c	2010-11-27 15:30:17.038757349 +0100
@@ -333,7 +333,7 @@
 	pid_t pid, sid;
 
     /* already a daemon */
-    if ( getppid() == 1 ) return 0;
+    //if ( getppid() == 1 ) return 0;
 
 	/* fork off the parent process */
 	pid = fork();
